### DISCUSSION TITLE
新支援卡池抽卡报错

### DISCUSSION THREAD

**Discussion by kaskadey** [azmiao/uma_plugin#48] [General]
![image](https://user-images.githubusercontent.com/116192167/196744723-90a16cc9-b7d0-4624-8401-8ca5fd30cbb3.png)

日志如上，抽井时出现错误。目测好像是新UP的卡和马出错？

**Comment by azmiao** [comment]
在排查了，另外报错反馈请反馈至issue处

**Reply by kaskadey** [reply]
感谢大佬解答，下次一定注意！

**Comment by azmiao** [comment]
看了下，昨晚我更新的bwiki时候，忘记刷新了，所以那边数据还是旧版，现在我刷新了一下，你可以在群里发一句“更新马娘卡池”，即可使用
